homeassistant/data_entry_flow.py

```diff
@@ -449,11 +447,14 @@ async def _async_setup_preview(self, flow: FlowHandler) -> None:
             await flow.async_setup_preview(self.hass)
 
 
-class FlowHandler:
+_FlowResultT = TypeVar("_FlowResultT", bound="FlowResult")
```

Comment:
Move this to just after the definition of `FlowResult` to make it easier to find